.github/workflows/main.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - main
       - inspector-*.*.x
+      - feature/0.2.0-SNAPSHOT
```

Comment:
Should it not be an inspector-0.2.0 branch, so that the branch matches?

Reply:
The branch name matches pefectly. The `feature/0.2.0-SNAPSHOT` is the one with AxonIQ console development currently

Reply:
Yes, but it would be more in line with the rest if it were `inspector-0.2.x` and it doesn't need the additional branch in that case.